ow_power_system/config/system.cfg

```diff
@@ -28,16 +28,21 @@ voltage_range: 0.1			        # volts
 # values from other planetary rovers, and some are just guesses.
 
 # Baseline values (watts) for constant power draws.
-baseline_power_camera: 0.1
-baseline_power_communications: 0.6
 baseline_power_computing: 0.1
 baseline_power_heating: 1.2
 baseline_power_science_instr: 0.6
 baseline_power_sample_handling: 0.6
 baseline_power_other: 2.2
 
+
 # Power coefficients for dynamically computed power draw.
-wattage_coefficient_lights: 0.3    # power draw (W) per light at max intensity
+power_lights_coefficient: 0.3       # power draw (W) per light at max intensity
+power_camera_controller: 1.0        # power draw of the camera controller
+power_camera_exposed_coefficient: 2.0  # power draw while a camera is exposing
+power_comms_active: 28.0           # power draw when comms is activated
+
+# length of time the antenna draws power during an ActivateComms operation
+communication_duration: 10.0       # seconds
```

Comment:
If the value here takes that much real time, I suggest making the default smaller, like 2-5 seconds, to lessen overall simulation time.  It would be good to find out what the average comms time is, though not hold up the PR over this.

Reply:
Since we don't have a good intuition or any data on this, but I think comms times could vary a lot, I'd like to request that the duration (in seconds)  be made a parameter (goal) of the action.  I'm assuming this is easy, and would make the action a lot more flexible.

Reply:
Will make communication duration a parameter in the ActivateComms action

Reply:
Done